README.md

```diff
@@ -28,9 +29,9 @@ Branching Strategy and Timelines
 
 | Yocto Release | IoT Edge version | Branch Name | Branch Status |
 | :- | :- | :- | :- |
-| Scrathgap | 1.5.x | scarthgap | TBD |
-| Kirkstone | 1.5.x | kirkstone | TBD |
-| Kirkstone | 1.4.x | main | Active and maintained |
+| Scrathgap | 1.5.x | main | Active and maintained |
+| Kirkstone | 1.5.x | kirkstone | Active and maintained |
+| Kirkstone | 1.4.x | kirkstone | Out of Support Nov'2024 |
```

Comment:
Are both 1.4 and 1.5 maintained in the same branch?

Reply:
ya we have been replacing the recipes to latest as we talked about. Since 1.4.x will go out of support I want to point it pit clearly